pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -119,6 +119,11 @@ with stdenv.lib;
     B43_PHY_HT? y
   ''}
   BCMA_HOST_PCI? y
+  ${if versionAtLeast version "3.17" then ''
+    BRCMFMAC_SDIO y
+    BRCMFMAC_USB y
+    BRCMFMAC_PCIE y
+  ''}
```

Comment:
Implementation note: `if` without `else` part won't work, and it's probably better to use `optionalString` anyway like in the code around.